README.md

```diff
@@ -1,38 +1,74 @@
-## Description
+[Capitaine Train](https://www.capitainetrain.com) sells tickets for numerous train operators across Europe.
 
-Capitaine Train sells train tickets for many operators across Europe.
+## Stations - A Database of European Train Stations
 
-This file contains all informations we use to identify a station in any system of the carriers.
+This repository contains the information Capitaine Train uses to identify stations across the diverse systems of the train operators.
 
-If you want to contribute to this database, please read `CONTRIBUTING.md`.
+If you want to contribute to this database, please read [CONTRIBUTING.md](https://github.com/capitainetrain/stations/blob/master/CONTRIBUTING.md).
 
 ## Licence
 
-The file `stations.csv` is distributed under the ODbL licence (see `LICENCE.txt`). In short,
-any modification to this data source must be published.
+The file `stations.csv` is distributed under the Open Database License (ODbL) licence, see [LICENCE.txt](https://github.com/capitainetrain/stations/blob/master/LICENCE.txt). In short, any modification to this data source must be published.
 
-Data sources include [Openstreetmap](http://www.openstreetmap.org) and [SNCF OpenData](https://ressources.data.sncf.com/explore/dataset/referentiel-gares-voyageurs/) that are under ODbL or compatible and a lot of sweat, tears and blood.
+Data sources include [OpenStreetMap](https://www.openstreetmap.org) and [SNCF OpenData](https://ressources.data.sncf.com/explore/dataset/referentiel-gares-voyageurs/), both released under ODbL or compatible licences; and a lot of blood, sweat, and tears.
```

Comment:
Fun to see the original phrasing. Common English use today is **blood, sweat, and tears** with the original sounding unusual to a native English speaker.